py/picca/tests/delta_extraction/expected_flux_tests.py

```diff
@@ -805,6 +812,442 @@ def test_expected_flux(self):
+                }})
+        for key, value in defaults_true_continuum.items():
+            if key not in config["expected flux"]:
+                config["expected flux"][key] = str(value)
+        # this should raise an error as iter out prefix should not have a folder
+        with self.assertRaises(ExpectedFluxError):
+            expected_flux = TrueContinuum(config["expected flux"])
+
+        config = ConfigParser()
+        config.read_dict(
+            {"expected flux": {
+                "input directory": f"{THIS_DIR}/data",
+                "iter out prefix": "iter_out_prefix",
+                "out dir": f"{THIS_DIR}/results/",
+            }})
+        for key, value in defaults_true_continuum.items():
+            if key not in config["expected flux"]:
+                config["expected flux"][key] = str(value)
+        # this should also raise an error as Forest variables are not defined
+        with self.assertRaises(ExpectedFluxError):
+            expected_flux = TrueContinuum(config["expected flux"])
+
+        # setup Forest variables; case: logarithmic wavelength solution
+        setup_forest("log", rebin=3)
+        expected_flux = TrueContinuum(config["expected flux"])
+
+        self.assertTrue(isinstance(expected_flux.get_var_lss, interp1d))
+
+        # setup Forest variables; case: linear wavelength solution
+        reset_forest()
+        setup_forest("lin", pixel_step=2.4)
+        expected_flux = TrueContinuum(config["expected flux"])
+
+        self.assertTrue(isinstance(expected_flux.get_var_lss, interp1d))
+
+        # Assert invalid binning raises ExpectedFluxError
+        with self.assertRaises(ExpectedFluxError):
+            setup_forest("lin", pixel_step=0.4)
+            expected_flux = TrueContinuum(config["expected flux"])
+
+    def test_true_continuum_read_raw_statistics(self):
+        """Test reading raw statistics files"""
+        # setup Forest variables; case: linear wavelength solution
+        setup_forest("lin", pixel_step=2.4)
+
+        data_dir = Path(__file__).parent / "data"        
+        
+        # initialize Data and Dr16ExpectedFlux instances
+        config = ConfigParser()
+        config.read_dict({
+            "data": desi_mock_data_kwargs,
+            "expected flux": {
+                "type": "TrueContinuum",
+                "input directory": f"{THIS_DIR}/data/data",
+                "iter out prefix": "iter_out_prefix",
+                "out dir": f"{THIS_DIR}/results/",
+            },
+        })
+
+        for key, value in defaults_true_continuum.items():
+            if key not in config["expected flux"]:
+                config["expected flux"][key] = str(value)
+        for key, value in defaults_desisim_data.items():
+            if key not in config["data"]:
+                config["data"][key] = str(value)
+        expected_flux = TrueContinuum(config["expected flux"])
+
+        var_lss = expected_flux.get_var_lss(10**Forest.log_lambda_grid)[:-1]
```

Comment:
I would get the `log_lambda_grid` from the test file. This way if we change the default values for this variable (in `setup_forest`) the test will still work

Reply:
Also this means we can drop the `Forest` import at the top